src/fem/dh_cell_accessor.hh

```diff
@@ -53,7 +53,7 @@ public:
 
     /// Return local index to element (index of DOF handler).
     inline unsigned int local_idx() const {
-    	ASSERT_LT_DBG(loc_ele_idx_, dof_handler_->el_ds_->lsize()+dof_handler_->ghost_4_loc.size()).error("Local element index is out of range!\n");
+        ASSERT_LT_DBG(loc_ele_idx_, dof_handler_->global_to_local_el_idx_.size()).error("Local element index is out of range!\n");
```

Comment:
If we want to use DHCellAccessor::get_dof_indices() for sequential DH, then we need to define local_idx() for all elements. So now we take local_idx() as the index in the map global_to_local_el_idx_.

Reply:
Do you mean sequential DH but parallel mesh? Then yes. But that should be communicated by sequential() method isn't it? Anyway I hope that finally the sequential DH will not used.